### DISCUSSION TITLE
Feature Request: Status information for Scrapers

### DISCUSSION THREAD

**Discussion by BigHoss** [grafana/grafana#51738] [Data Sources]
It would be really helpful to see the status of a scraper like in prometheus.

Suggestions for the values
- Status of the endpoint (up/down)
- Interval
- Amount of scraped values

**Comment by BigHoss** [comment]
Wrong repo 🤦‍♂️

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
